### DISCUSSION TITLE
Introduce Path Styles

### DISCUSSION THREAD

**Discussion by karst** [OpenRCT2/OpenRCT2#20383] [Feature requests]
This topic has come up quite a few times on discord, so I am surprised that this isn't a topic yet in the discussions.
It would be great if all path surfaces would be able to swap between some versions so you can mix and match behaviours.
If the sprites are available for it then allow the following:
- Swap between hill styles: Ramp or Stairs
- Swap between corner styles: Rounded, Squared or Diagonal

Some of these functions would also be useful for the soaked project.

**Comment by 733737** [comment]
I think @ZeeMaji already has this prototyped

**Reply by ZeeMaji** [reply]
Er, i never prototyped such a thing. I was working on some icons that are somewhat related to this but i never finished them and path styles would allow for better icons anyhow. (i imagine they would use the empty space for a 4th icon in the path tool)

**Comment by TheClaudeQC** [comment]
that could be a great idea so I'm not mix up whit the grey, brown and red tamacs & brown and black dirts anymore

**Comment by J-PIE-314** [comment]
#17760 suggests adding color as a type of path style.

**Comment by J-PIE-314** [comment]
#16035 is a similar request to this one.
